packages/components/stamp/src/stamp.tsx

```diff
@@ -20,11 +24,75 @@ type Props = {
    * If `true`, renders a condensed version of the stamp.
    */
   isCondensed: boolean;
-  children: ReactNode;
+  children?: ReactNode;
+  icon?: ReactElement;
+  message?: MessageDescriptor & {
```

Comment:
Usually we name this prop `intlMessage`. Any reason for naming it `message`?

Reply:
Also, I think we need to add a new prop here. The `intlMessage` prop is usually an alternative to another prop that can be used if you don't use messages.

If we are deprecating `children`, we should add something like `label` (which is a `string`) to accept the text value. Then the user should chose between `label` or `intlMessage`.

Reply:
Super useful hint about the purpose of `intlMessage` 🙇 

I was wondering though about what you suggest of having both `label` and `intlMessage`. It seems the stamp it's about rendering text (and an optional icon) so I would expect (as a consumer) to only have a (maybe) `label` property which could be a `string` (in case you don't use translations) or a `MessageDescriptor`.
Aren't we supposed to have such typing for a single property?

Reply:
+1 to have only the `label` prop as a _string_.

After all, we want to control the typography from the component so it's perhaps better to now allow `Text.*` components.

Thanks!

Reply:
Good points. So what we are suggesting is the trade-off is worth it, that is no translations. Just use the label property to pass in string text only.

Reply:
Actually, what we suggest is to only have the `label` property with the _string_ type, but it's up to consumers to provide a translated message. The could do it using the `useIntl` hook.
For instance:
```jsx
const const intl = useIntl();

return (
  <Stamp label={intl.formatMessage(messages.ssoConfigStatusLabel)} />
);
```

Reply:
Ah, I get it now. that makes sense. I will update it now

Reply:
Updated here: https://github.com/commercetools/ui-kit/pull/2383/commits/05a1ecd9e5cc1d174b6efa700f439ad63c10a264

Thank you